tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -482,6 +460,559 @@ There are <?php echo $variable; ?> examples
+<?php echo '2';
+
+`;
+
+exports[`mixed-13.php 1`] = `
+<?php echo '1'; ?>
+<?php echo '2'; ?>
+<?php echo '3'; ?>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php echo '1'; ?>
+<?php echo '2'; ?>
+<?php echo '3'; ?>
+
+`;
+
+exports[`mixed-14.php 1`] = `
+<?php echo '1'; ?>
+<?php echo '2'; ?>
+<?php echo '3';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php echo '1'; ?>
+<?php echo '2'; ?>
+<?php echo '3';
+
+`;
+
+exports[`mixed-15.php 1`] = `
+<?php
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+
+`;
+
+exports[`mixed-16.php 1`] = `
+<section class="<?= $sectionClass ?>">
+  <header class="<?= $headerClass ?>">
+  <?php echo $someOtherVar()->element('collection/item-title', [
+      'class' => $itemTitleClass,
+      'small_title' => $item->small_title,
+      'title' => $item->title,
+      'url' => $url,
+      'color' => $item->title_color ?: 'primary',
+  ], [
+      'foo' => 'bar',
+      'bar' => 'foo',
+      'barfoo' => 'foobar',
+      'foobar' => 'barfoo'
+  ]); ?>
+  </header>
+  <div> <?php echo $someOtherVar()->element('collection/item-title', [
+      'class' => $itemTitleClass,
+      'small_title' => $item->small_title,
+      'title' => $item->title,
+      'url' => $url,
+      'color' => $item->title_color ?: 'primary',
+  ], [
+      'foo' => 'bar',
+      'bar' => 'foo',
+      'barfoo' => 'foobar',
+      'foobar' => 'barfoo'
+  ]); ?>
+  </div>
+</section>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<section class="<?= $sectionClass; ?>">
+  <header class="<?= $headerClass; ?>">
+  <?php echo $someOtherVar()->element(
+      'collection/item-title',
+      [
+          'class' => $itemTitleClass,
+          'small_title' => $item->small_title,
+          'title' => $item->title,
+          'url' => $url,
+          'color' => $item->title_color ?: 'primary'
+      ],
+      [
+          'foo' => 'bar',
+          'bar' => 'foo',
+          'barfoo' => 'foobar',
+          'foobar' => 'barfoo'
+      ]
+  ); ?>
+  </header>
+  <div> <?php echo $someOtherVar()->element(
+      'collection/item-title',
+      [
+          'class' => $itemTitleClass,
+          'small_title' => $item->small_title,
+          'title' => $item->title,
+          'url' => $url,
+          'color' => $item->title_color ?: 'primary'
+      ],
+      [
+          'foo' => 'bar',
+          'bar' => 'foo',
+          'barfoo' => 'foobar',
+          'foobar' => 'barfoo'
+      ]
+  ); ?>
+  </div>
+</section>
+
+`;
+
+exports[`mixed-17.php 1`] = `
+<div>
+    <?php switch($variable) {
+    case 1: ?>
+        <div>Newspage</div>
+        <?php break; ?>
+    <?php case 2: ?>
+    <div>Forum</div>
+    <?php break; ?>
+    <?php } ?>
+</div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+    <?php switch ($variable) { case 1: ?>
```

Comment:
The original version with the linebreak looks better, doesn't it?

Reply:
@czosel `switch` is very difficult print with inline nodes, current output same as was, i think we can do this in other PR, because he is already big

Reply:
it's not just switch though right? there are a bunch of examples above (`echo`, `function`) that should break as well

Reply:
I agree switch looks some ugly, let's improve output control structures in other PR, they require logic inside `node`, not only in `printLines`. Main purpose this PR is solve problem with `<?php ... ?><?php ... ?>` and indenting multiple statements inside `<?php ... ?>`.

Also i will try to solve in this PR break multiple statement.